recipes/raylib/all/conanfile.py

```diff
@@ -22,11 +23,21 @@ class RaylibConan(ConanFile):
         "shared": [True, False],
         "fPIC": [True, False],
         "opengl_version": [None, "4.3", "3.3", "2.1", "1.1", "ES-2.0"],
+
+        "module_raudio": [True, False],
+
+        "events_waiting": [True, False],
+        "custom_frame_control": [True, False]
```

Comment:
```suggestion
        "customize_build": [True, False],
        "module_raudio": [True, False],
        "events_waiting": [True, False],
        "custom_frame_control": [True, False]
```

Add customize_build as option, as it may affect other options too.